<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowResults</class>
 <widget class="QDialog" name="ShowResults">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zobrazit výsledky</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>0</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Poslední výsledek</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
